packages/lib/src/components/PayPal/Paypal.tsx

```diff
@@ -31,8 +35,11 @@ class PaypalElement extends UIElement<PayPalElementProps> {
         const intent: Intent = isZeroAuth ? 'tokenize' : props.intent || intentFromConfig;
         const vault = intent === 'tokenize' || props.vault;
 
+        const displayContinueToReviewPageButton = props.userAction === 'continue';
+
         return {
             ...props,
+            commit: displayContinueToReviewPageButton ? false : props.commit,
```

Comment:
Would it be clearer to pass the `displayContinueToReviewPageButton` through to the component as a boolean rather than having to re-evaluate/re-create `displayContinueToReviewPageButton` in `PayPalButtons.tsx`?

Reply:
We can. I wanted to avoid more prop drilling but it is ok to do that here